### PR TITLE
BugReport queue function

### PR DESCRIPTION
**Pull request by davidreneuw**
Created a BugReport Queue trigger function that creates ADO issues and emails Data Solutions email.
Created an EmailService that makes building emails easier for functions
Changed all functions to use the EmailService
Moved the email templates to a different folder
Created BugReportMessage and Sender so that Mediator can handle bug reports
Implemented BugReports in the HelpDialog and on System Error pages.
Unit tested the BugReport queue function and the EmailService.

### CODE REVIEW

File: Portal/src/Datahub.Functions/EmailTemplates/bug_report.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+﻿Bug Report - {title}
+<p>
+    A bug report was submitted and issue {id} was created on <a href="{url}">Azure DevOps</a>. Description: {description}
+</p>
+<p>
+    Un rapport de bogue a été soumis et l'issue {id} a été créée sur <a href="{url}">Azure DevOps</a>. Description: {description}
```
Comment (ErikApption) [inline Portal/src/Datahub.Functions/EmailTemplates/bug_report.html]: Un rapport de bogue a ete soumis et le problème/ticket {id} a ete cree

File: Portal/src/Datahub.Functions/Services/EmailService.cs
Comment (ErikApption) [inline Portal/src/Datahub.Functions/Services/EmailService.cs:1]: haven't done any other digging in code but didn't we already have an email service?

**Review by ErikApption — CHANGES_REQUESTED** [review]
Can you confirm that there is no duplication in emailservice in function and there is a minor suggestion for French translation in my comment

File: Portal/src/Datahub.Functions/Services/EmailService.cs
Comment (davidreneuw) [reply Portal/src/Datahub.Functions/Services/EmailService.cs:1]: @ErikApption There is an email service in the Infrastructure project, the service I built could very well be merged into it. I just was not sure where to build the service (both interface and service inside Functions project v.s. interface in Application project and service in Infrastructure project, at which point I would have merged them)

**Comment by davidreneuw** [comment]
> Can you confirm that there is no duplication in emailservice in function and there is a minor suggestion for French translation in my comment

I confirm there is no duplication between the EmailService I built and the two existing email services i.e. DatahubEmailService (in Infrastructure) and EmailNotificationService (in Core). Although they could probably be merged.

File: Portal/src/Datahub.Functions/Services/EmailService.cs
Comment (ErikApption) [reply Portal/src/Datahub.Functions/Services/EmailService.cs:1]: makes sense, lets figure this one later once Yannick is back

**Pull request merged**